runbotics/runbotics-actions-windows/src/automation/office/excel.action-handler.ts

```diff
@@ -259,6 +260,27 @@ export default class ExcelActionHandler extends StatefulActionHandler {
         }
     }
 
+    async insertRowsAfter(
+        input: ExcelInsertRowsActionInput
+    ): Promise<void> {
+        const targetWorksheet = this.session.Worksheets(input?.worksheet ?? this.session.ActiveSheet.Name);
+        
+        try {
+            const startingRow = input.startingRow;
+            const rowsNumber = input.rowsNumber
+
+            if (rowsNumber <= 0) throw new Error();
+
+            targetWorksheet
+                .Range(
+                    targetWorksheet.Rows(startingRow + 1),
+                    targetWorksheet.Rows(startingRow + rowsNumber))
+                .Insert();
+        } catch (e) {
+            throw new Error(ExcelErrorMessage.insertRowsIncorrectInput(e));
+        }
+    }
```

Comment:
Is this try catch necessary? You could just throw your desired exception in if above.

Reply:
also imo here it should be solved without try and catch, just by checking rowNumber and startingRow, an error could be thrown to clearly inform what we expect from user before calling the Excel method. what also speaks for this solution is that these errors thrown strictly from Excel methods are non-descriptive and actually cause more confusion, as you showed us on ss. perhaps the rest of the actions will be refactored in the future in this regard

Reply:
Agree! Changes to guards.